form/data_transformers.rst

```diff
@@ -80,6 +80,7 @@ class::
                 ->addModelTransformer(new CallbackTransformer(
                     function ($tagsAsArray) {
                         // transform the array to a string
+                        $tagsAsArray = ['tag1', 'tag2', 'tag3'];
```

Comment:
I don't use transformers much ... but the current code looks OK to me ... because `$tagsAsArray` is passed as an argument of the callback, so it would be defined in runtime.

Reply:
Thank you @javiereguiluz 

I understand well, but if you take the code and paste it for testing, the first test will never work.

So I thought of adding this line, but if you judge the opposite. Its good

Thank you

Reply:
The value should not be hardcoded here, it's up to the test to define some context before expected result.

Reply:
I agree, if we overwrite the value each time this function is called, this will end up always returning `tag1,tag2,tag3` no matter what array was passed.

If you mean that there are problems when no tag is parsed (because `$tagsAsArray` is then null or the like, I don't know?), then we should probably add something like (`$tagsAsArray = is_array($tags) ? $tags : [];`).

Reply:
I understand that the table must be completed before, but the example given poses a problem, so at least the tests must be reinforced

So if we add the code propesed by @wouterj  that solves the problem

I propose (PHP 7) : $tagsAsArray = is_array($tagsAsArray) ?: [];